dist/api_data.py

```diff
@@ -1099,6 +1099,22 @@ def __ge__(self, other):
     ]),
 ]
 
+# At this stage live restore intentionally does not support reconfiguring the number of worker
+# threads. If that becomes necessary in the future we'll need to break out the thread count config
+# and add it to the reconfigure items too.
```

Comment:
Since we're giving reconfigure advice in this comment please add something about adding a MAX_WORKER sort of macro and using that maximum value when adding in the internal sessions needed in the connection.

Reply:
d38f4c8b1